code/src/components/TaskCard.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import DayJS from 'react-dayjs';
+import styled from 'styled-components';
+
+const Container = styled.div`
+margin: 20px 0;
+padding: 10px 15px 5px;
+display; flex;
+flex-direction: column;
+justify-content: space-between;
+align-items: center;
+background-color: white;
+border-radius: 15px;
+box-shadow: 0  0 10px #aaaaaa;
+@media (min-width: 768px){
+  width: 680px;
+  padding: 15px 20px 10px;
+}
+`;
+const UpperRow = styled.p`
+width: 100%;
+margin: 0;
+display: flex;
+flex-direction: row;
+justify-content: space-between;
+align-items: center;
+@media (min-width: 768px){
+  width: 600px;
+}
+`;
+const Label = styled.label`
+display: inline-block;
+position: relative;
+padding-left: 35px;
+margin: 0px;
```

Comment:
px could be removed for 0 margin values ?

Reply:
Fixed